cmake/VISPUtils.cmake

```diff
@@ -1263,6 +1263,13 @@ macro(vp_get_version_from_pkg LIBNAME PKG_PATH OUTPUT_VAR)
   endif()
 endmacro()
 
+macro(vp_get_version_from_pkg2 LIBNAME PKG_PATH OUTPUT_VAR)
```

Comment:
Quick solution to parse the OpenBLAS version. Difference is between `version=` and `Version: ` styles.